pkg/services/object/get/exec.go

```diff
@@ -322,9 +374,29 @@ func (exec *execCtx) writeObjectPayload(obj *objectSDK.Object) bool {
 	return err == nil
 }
 
+// writeFromStream writes payload from stream.
+func (exec *execCtx) writeFromStream(reader io.ReadCloser) error {
```

Comment:
```suggestion
func copyPayloadStream(w ChunkWriter, r io.Reader) error
```
would be more reusable